File: section6#Color/C++/color.cpp
#include "pch.h"
#include <opencv2/opencv.hpp>
#include <iostream>

using namespace std;
using namespace cv;

int main()
{
	Mat img = imread("check-circle.png");	//Put Girl.jpg into the same path with source code.
	if (img.empty())
	{
		cout << "The picture is not exist!" << endl;
		return 1;
	}
	namedWindow("Girl");		//Windowns named Girl
	imshow("Girl", img);		//Show img in Girl window

	Mat gray;
	cvtColor(img, gray, COLOR_BGR2GRAY);
	namedWindow("GirlGray");
	imshow("GirlGray", gray);
	imwrite("check-circle_gray.jpg", gray);

	waitKey(6000);			//Waiting for a key input or 6S timeout
	return 0;
}

File: section6#Color/Python/color.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
# author：lowkeyway time:7/1/2019

import sys
import cv2 as cv

def main_func(argv):
    img = cv.imread("girl.jpg")
    cv.namedWindow("Girl")
    cv.imshow("Girl", img)

    gray = cv.cvtColor(img, cv.COLOR_BGR2GRAY)
    cv.namedWindow("GirlGray")
    cv.imshow("GirlGray", gray)
    cv.imwrite("GirlGray.jpg", gray)

    cv.waitKey(6000)
    cv.destroyAllWindows()

if __name__ == '__main__':
    main_func(sys.argv)